README.md

```diff
@@ -37,23 +37,35 @@ import { WebSocket, Server } from 'mock-socket';
 ```js
 import test from 'ava';
 import { Server } from 'mock-socket';
+import {AsyncQueue} from '@borewit/async-queue';
```

Comment:
I'd like to keep the readme simple and avoid using 3rd party libs

Reply:
I fully understand that. I tried to embed an inline solution, yet that started to a bit lengthy and distracting, hence I moved this portion to  [borewit/async-queue](https://github.com/Borewit/async-queue).

I fixed the code example in 3 different area's. I leave it as is.